README.md

```diff
@@ -153,6 +153,13 @@ feed:
     - updates
 ```
 
+Or, to generate a feed for all categories:
+
+```yml
+feed:
+  categories: true
```

Comment:
On line 24 of generator.rb, we have special handling for `config["tags"]`. Do we intend to only generate collection-level categories?